src/index.ts

```diff
@@ -84,10 +85,10 @@ export class SilentPayment {
     for (const group of silentPaymentGroups) {
       // Bscan * a * outpoint_hash
       const ecdh_shared_secret_step1 = Buffer.from(ecc.privateMultiply(outpoint_hash, a) as Uint8Array);
-      const ecdh_shared_secret = ecc.pointMultiply(group.Bscan, ecdh_shared_secret_step1);
+      const ecdh_shared_secret = Buffer.from(ecc.getSharedSecret(ecdh_shared_secret_step1, group.Bscan) as Uint8Array);
```

Comment:
I'm curious, how is this better?

Reply:
Currently, it doesn't change anything since `getSharedSecret` is calling `Point.multiply` under the hood (which was surprising to me). But ideally, `getSharedSecret` should be doing a constant time multiplication. If `getSharedSecret` is updated in the future to do constant time multiplication, we won't have to change anything here. I seem to remember seeing a todo in the noble crypto repos about changing this to constant time, but will need to go back and find it.